### PR TITLE
G: DELETE: can submit posts and DELETE them

### PR DESCRIPTION
**Pull request by Edward-Phillips**
added delete route and button to each post to delete them, a pop up should appear asking if you are sure.

### CODE REVIEW

**Comment by FayeCarter** [comment]
Love it

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -2,6 +2,9 @@
   <ul class="post">
     <li><%= post.formatted_time %></li>
     <li class="message"><%= post.message %></li>
+    <li><%=link_to 'Delete', post_path(post),
+    method: :delete,
+    data: { confirm: 'Are you sure?'} %></li>
```
Comment (tommyrharper) [inline app/views/posts/index.html.erb:7]: Good stuff, making sure we don't delete by accident.

File: app/controllers/posts_controller.rb [hunk 2]
```diff
@@ -12,6 +12,12 @@ def index
     @posts = Post.all.reverse
   end
 
+  def destroy
+    @post = Post.find(params[:id])
+    @post.destroy
+    redirect_to posts_path
+  end
```
Comment (FayeCarter) [inline app/controllers/posts_controller.rb:19]: game changer

File: app/views/posts/index.html.erb (hunk 1 above)
Comment (FayeCarter) [inline app/views/posts/index.html.erb:7]: stunning

**Pull request merged**